src/api-reference/direct-connects/hotel-service-2/xsd/HotelService2.xsd

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<?xml-stylesheet type="text/xsl" href="xs3p_better_doc.xsl"?>
```

Comment:
This line is preventing https://developer.concur.com/api-reference/direct-connects/hotel-service-2/xsd/HotelService2.xsd from rendering as xs3p_better_doc.xsl doesn't exist.